frontend/src/metabase/browse/components/MetricsTable.tsx

```diff
@@ -293,3 +321,107 @@ function DescriptionCell({ metric }: { metric?: MetricResult }) {
     </Cell>
   );
 }
+
+type MetricAction = {
+  key: string;
+  title: string;
+  icon: IconName;
+  action: () => void;
+};
+
+function MenuCell({ metric }: { metric?: MetricResult }) {
+  const [createBookmark] = useCreateBookmarkMutation();
```

Comment:
If you use RTK query directly you would need to invalidate the entity framework cache manually otherwise the list of bookmarks in the sidebar is not updated:
```
dispatch(Bookmarks.actions.invalidateLists())
```